index.html

```diff
@@ -7,22 +7,22 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
     <link rel="stylesheet" href="style.css">
-    <title>My Personal Portifolio</title>
+    <title>Ian Kiema Portifolio</title>
 </head>
 <body>
```

Comment:
:red_circle: [REQUIRED] I ran a contrast test using  [tool](https://www.aditus.io/button-contrast-checker/)  on your portfolio and found some areas where the color contrast could be improved. Please [follow this link](https://www.aditus.io/button-contrast-checker/cdn-githubraw-com-2023-06-19-at-19-06-12-995?failing) to see the result 

Below is just a screenshot
![Firefox_Screenshot_2023-06-19T19-20-04 818Z](https://github.com/iankiema/My-Portifolio-Personal/assets/26736582/338f2ae9-e558-4aee-86ef-115d43431135)